src/styles/builder.scss

```diff
@@ -142,14 +146,14 @@ $headerHeight: 170px;
     height: 100%;
     min-height: 100vh;
     position: absolute;
-    width: 100%;
+    width: calc(100% - #{$offcanvas});
```

Comment:
I think we cant use this here or we can use it in big viewports.
This application was thought to be a mobile website and having a limited width we need to do some assumptions like the offcanvas effect
![image](https://user-images.githubusercontent.com/6814061/80922972-e07f7080-8d80-11ea-9d6b-d903894e947a.png)
I think your approach is good but applied to >= tablet devices :D

Reply:
Agree. Shall we create an issue for implementing Media Queries?

Reply:
https://github.com/inspiratweb/Diet/pull/7 :D